src/d3d9/d3d9_device.cpp

```diff
@@ -936,6 +936,9 @@ namespace dxvk {
     const VkImageSubresource dstSubresource = dstTextureInfo->GetSubresourceFromIndex(dstFormatInfo->aspectMask, dst->GetSubresource());
     const VkImageSubresource srcSubresource = srcTextureInfo->GetSubresourceFromIndex(srcFormatInfo->aspectMask, src->GetSubresource());
 
+    if (unlikely((srcSubresource.aspectMask & (VK_IMAGE_ASPECT_DEPTH_BIT | VK_IMAGE_ASPECT_STENCIL_BIT)) && m_activeBeginScene))
```

Comment:
I'm not quite up to date, does StretchRect work on DS textures in DXVK? If not, you should add a TODO comment/warning while you're at it.

Reply:
It seems to handle resolve/copy, the blit path is a bit questionable.